src/device_trezor/CMakeLists.txt

```diff
@@ -63,37 +63,22 @@ set(trezor_sources
 set(trezor_private_headers)
 
 
-include(FindProtobuf)
-find_package(Protobuf)  # REQUIRED
-
-# Test for HAVE_PROTOBUF from the parent
-if(Protobuf_FOUND AND HAVE_PROTOBUF)
-    if ("$ENV{PYTHON3}" STREQUAL "")
-        set(PYTHON3 "python3")
-    else()
-        set(PYTHON3 "$ENV{PYTHON3}" CACHE INTERNAL "Copied from environment variable")
-    endif()
-
-    execute_process(COMMAND ${PYTHON3} tools/build_protob.py WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/trezor RESULT_VARIABLE RET OUTPUT_VARIABLE OUT ERROR_VARIABLE ERR)
-    if(RET)
-        message(WARNING "Trezor protobuf messages could not be regenerated (err=${RET}, python ${PYTHON})."
-                        "OUT: ${OUT}, ERR: ${ERR}."
-                        "Please read src/device_trezor/trezor/tools/README.md")
-    else()
-        message(STATUS "Trezor protobuf messages regenerated ${OUT}")
-        set(TREZOR_PROTOBUF_GENERATED 1)
-    endif()
-endif()
-
-
-if(TREZOR_PROTOBUF_GENERATED)
+# Protobuf and LibUSB processed by CheckTrezor
+if(DEVICE_TREZOR_READY)
     message(STATUS "Trezor support enabled")
 
     add_definitions(-DPROTOBUF_INLINE_NOT_IN_HEADERS=0)
+    set ( libdevice_trezor_hdr "" )
+    set ( libdevice_trezor_lib "" )
+
+    if(LibUSB_COMPILE_TEST_PASSED)
+        list(APPEND libdevice_trezor_hdr ${LibUSB_INCLUDE_DIRS})
+        list(APPEND libdevice_trezor_lib ${LibUSB_LIBRARIES})
```

Comment:
That seems like an odd name for the variable, referencing trezor when it's about libusb ?

Reply:
The idea of `libdevice_trezor_lib` and `libdevice_trezor_hdr` was to contain other optional libraries for Trezor but for now it contains only LibUSB. At this point, I can remove it completely and just use `${LibUSB_INCLUDE_DIRS}` and `${LibUSB_LIBRARIES}`